src/Components/Automotive/Automative.js

```diff
@@ -19,17 +19,21 @@ function AutomativeDropdown() {
   const handleChange = (event) => {
     setService(event.target.value);
   };
```

Comment:
improve on in-code documentation explaining each component why it may be commented out